lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  font-size: $nx-font-size;
+  max-width: $nx-sidebar-max-width;
```

Comment:
Does this and its corresponding variable do anything?  Seems redundant with the `width`.

Reply:
> Does this and its corresponding variable do anything? Seems redundant with the `width`.

The mock specifies a max-width of 300px. Should that be a modifier class instead?

Reply:
> The mock specifies a max-width of 300px. Should that be a modifier class instead?

Hmm I think that may be an inconsistency in the mock, or at any rate we need to clarify what is meant by it.  Since the width isn't dynamic at all there's not really a conceptual difference between the max width and the actual width, so we're basically getting two different numbers for the same thing here.  @dturner32 ?

Reply:
> > The mock specifies a max-width of 300px. Should that be a modifier class instead?
> 
> Hmm I think that may be an inconsistency in the mock, or at any rate we need to clarify what is meant by it. Since the width isn't dynamic at all there's not really a conceptual difference between the max width and the actual width, so we're basically getting two different numbers for the same thing here. @dturner32 ?

What I recall from our meeting regarding that 300px width is that it'll be used when the bar has a long product name (most likely true for Repository Manager).

Reply:
> What I recall from our meeting regarding that 300px width is that it'll be used when the bar has a long product name (most likely true for Repository Manager).

But in that case the product in question would have to increase the `width`.  That is true whether or not this `max-width` is set.  So the `max-width` is still not doing anything there.

Reply:
> > What I recall from our meeting regarding that 300px width is that it'll be used when the bar has a long product name (most likely true for Repository Manager).
> 
> But in that case the product in question would have to increase the `width`. That is true whether or not this `max-width` is set. So the `max-width` is still not doing anything there.

Removed in 5812b377c330f9377bd19a0c04752d6b9bacaecf